ext/BijectorsTapirExt.jl

```diff
@@ -0,0 +1,40 @@
+module BijectorsTapirExt
+
+if isdefined(Base, :get_extension)
+    using Tapir: @is_primitive, MinimalCtx, Tapir, CoDual, primal, tangent_type, @from_rrule
+    using Bijectors: find_alpha
+    using ChainRulesCore: rrule
```

Comment:
It's necessary (or at least recommended?) to not load any other packages apart from the weak dependency and the parent package: https://github.com/JuliaStats/LogExpFunctions.jl/pull/63#issue-1571562080
```suggestion
    using Bijectors: find_alpha, ChainRulesCore
```